source/scss/_library/_objects.text.scss

```diff
@@ -506,6 +506,12 @@ a.o-breadcrumb {
     position: absolute;
   }
 
+  .u-path-fill--white {
+    path {
```

Comment:
If we want this rule to support a wider range of svgs, we might consider a broader set of selectors here. We might not need this now though. I don't know enough about where this is used to know if this really matters. Up to you.
```
circle,ellipse,line,path,polygon,polyline,rect /* svg shapes*/ {
```

Reply:
Good point. I'll just target `svg *`.

Reply:
This has been updated.